lms/templates/courseware/news.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
Template contains a `${content}` which seems risky to escape.  You should check if this might contain HTML.
Reminder to search templates for `${...}`.

Reply:
Fixed and not escaping for now.